storage/debugbar/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Changes look good. No need to check into git anything from the storage/ folder however